src/Illuminate/Database/Query/Builder.php

```diff
@@ -2704,6 +2704,31 @@ public function orderBy($column, $direction = 'asc')
         return $this;
     }
 
+    /**
+     * Order the query by a column with a given set of priority values.
+     *
+     * @param  array{string|\BackedEnum}|null  $priorities
```

Comment:
This doesn't correctly reflect what you are doing within the method. The method also works with `UnitEnum`s

```suggestion
     * @param  array{string|\UnitEnum}|null  $priorities
```